tests/dm_control/test_dm_control_envs.py

```diff
@@ -0,0 +1,32 @@
+import pickle
+import unittest
+
+from dm_control.suite import ALL_TASKS
+import numpy as np
+
+from garage.envs.dm_control import DmControlEnv
+
+
+class TestDmControlEnvs(unittest.TestCase):
+    def test_dm_control_envs(self):
+        for task in ALL_TASKS:
+            env = DmControlEnv(domain_name=task[0], task_name=task[1])
+
+            # Test pickling
+            pickle.loads(pickle.dumps(env))
+
+            print("Testing", env.__class__)
+            ob_space = env.observation_space
+            act_space = env.action_space
+            ob = env.reset()
```

Comment:
Can you use the helper function we already have for this? Feel free to add your own asserts.